src/logging/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required (VERSION 3.13)
+
+project (yule VERSION 0.0.1 LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+add_compile_options(-O3 -std=c++17 -g -Werror -Wall -Wextra)
```

Comment:
Not a big deal but I don't think we should set -std here since we are telling cmake to do it above

Reply:
Yup. will remove